src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoRecipeWrapper.java

```diff
@@ -266,20 +294,21 @@ public void drawInfo(Minecraft minecraft, int recipeWidth, int recipeHeight, int
         this.lastMouseY = mouseY;
     }
 
-    private void drawText(Minecraft minecraft, int recipeWidth) {
+    @SideOnly(Side.CLIENT)
+    protected void drawHoveringInformationText(Minecraft minecraft, List<String> tooltip, int mouseX, int mouseY) {
+        int minX = recipeLayout.getRecipeCategory().getBackground().getWidth();
+        int[] yRange = new int[]{49, 69};
```

Comment:
More magic numbers type stuff going on here, and generally it's better to replace a complicated conditional expression with a function call so the intent of the expression is communicated. It's easy for logic errors to crop up in places like that because there's so much noise.